instructors.md

```diff
@@ -156,9 +199,143 @@ A typical schedule for a two-day course is:
     *   15:45 Open discussion
     *   16:15 Wrap up and minute cards for afternoon session
 
+## Why We're Not a MOOC
+
+> If you use robots to teach, you teach people to be robots.
+
+This difference between what novices are doing when they learn,
+and what competent practitioners are doing,
+is one of the reasons we have stopped trying to teach via recorded video
+with auto-graded drill exercises.
+Any recorded content is as ineffective for most learners as broadcast television,
+or as a professor standing in front of 400 people in a lecture hall,
+because neither can intervene to clear up specific learners' misconceptions.
+Some people happen to already have the right conceptual categories for a subject,
+or happen to form them correctly early on;
+these are the ones who stick with most massive online courses,
+but many discussions of the effectiveness of such courses
+ignore this survivor bias.
+
+## Why So Much Documentation Is Useless
+
+The transition from novice to competent practitioner is no more
+and no less than
+the construction of correct (enough) categories,
+i.e.,
+the construction of a new mental model of this new intellectual domain.
+The goal of education for novices is, therefore,
+to help them form the right categories.
+Until they've done that,
+trying to impart "mere information" just confuses them.
+
+This is one of the reasons software documentation is so often frustrating.
```

Comment:
The “frustrating” in this sentence is much less inflammatory than the “useless” in the section heading.  Since your point seems to be “it's hard/impossible to write one-size-fits-all-readers documentation”, not “those doc writers are just wasting their time”, I'd recommend the “frustrating” wording in the heading.